vm_helper_files/Makefile

```diff
@@ -17,6 +17,7 @@ help:
 	@echo "           server_version=<x.x.x>                        @param - required"
 	@echo "           cli_version=<x.x.x>                           @param - required"
 	@echo "  clean                                              Remove docker containers and images.  Also, qpc cli."
+	@echo "  install-cli                                        Install the qpc-tools cli locally."
```

Comment:
Let's call this `install-local-tools` and then later we could have `install-release-tools`